requirements-dev.txt

```diff
@@ -7,8 +7,9 @@ psutil>=5.8.0
 
 ## optional dependencies
 # ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
-ray[default]>=2.1.0,!=2.5.0
+ray>=2.1.0,!=2.5.0
```

Comment:
```suggestion
ray-core>=2.1.0,!=2.5.0
```
?

Reply:
Analogue of conda package `ray-core` for pip - just `ray`.